invenio_records_resources/resources/record.py

```diff
@@ -31,6 +34,26 @@ class RecordResourceConfig(ResourceConfig):
             RecordJSONSerializer(schema=RecordSchemaJSONV1)
         )
     }
+    error_map = {
+        InvalidQueryError: create_errormap_handler(
```

Comment:
moderate: is there a way of reusing the `description` of the base error. i.e. we are duplicating the string coming form other exceptions (e.g. `PermissionDeniedError`, etc.) would be nice to just reuse.